### ISSUE TITLE
Vagrant box not found

### ISSUE THREAD

**Issue by i11** [myoung34/vagrant-ovirt3#22]
```
$ curl -vv -L https://raw.github.com/myoung34/vagrant-ovirt3/master/example_box/ovirt.box
* Adding handle: conn: 0x7fb18200ba00
* Adding handle: send: 0
* Adding handle: recv: 0
* Curl_addHandleToPipeline: length: 1
* - Conn 0 (0x7fb18200ba00) send_pipe: 1, recv_pipe: 0
* About to connect() to raw.github.com port 443 (#0)
*   Trying 23.235.43.133...
* Connected to raw.github.com (23.235.43.133) port 443 (#0)
* TLS 1.2 connection using TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
* Server certificate: www.github.com
* Server certificate: DigiCert High Assurance CA-3
* Server certificate: DigiCert High Assurance EV Root CA
> GET /myoung34/vagrant-ovirt3/master/example_box/ovirt.box HTTP/1.1
> User-Agent: Mozilla/5.0 (compatible; MSIE 9.0; Windows NT 6.1; Trident/5.0)
> Host: raw.github.com
> Accept: */*
> Referer: 
> 
< HTTP/1.1 301 Moved Permanently
< Date: Tue, 13 Jan 2015 10:42:22 GMT
* Server Apache is not blacklisted
< Server: Apache
< Location: https://raw.githubusercontent.com/myoung34/vagrant-ovirt3/master/example_box/ovirt.box
< Content-Length: 0
< Accept-Ranges: bytes
< Via: 1.1 varnish
< Age: 0
< X-Served-By: cache-ams4121-AMS
< X-Cache: MISS
< X-Cache-Hits: 0
< Vary: Accept-Encoding
< 
* Connection #0 to host raw.github.com left intact
* Issue another request to this URL: 'https://raw.githubusercontent.com/myoung34/vagrant-ovirt3/master/example_box/ovirt.box'
* Adding handle: conn: 0x7fb18280b400
* Adding handle: send: 0
* Adding handle: recv: 0
* Curl_addHandleToPipeline: length: 1
* - Conn 1 (0x7fb18280b400) send_pipe: 1, recv_pipe: 0
* About to connect() to raw.githubusercontent.com port 443 (#1)
*   Trying 23.235.43.133...
* Connected to raw.githubusercontent.com (23.235.43.133) port 443 (#1)
* TLS 1.2 connection using TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA256
* Server certificate: www.github.com
* Server certificate: DigiCert High Assurance CA-3
* Server certificate: DigiCert High Assurance EV Root CA
> GET /myoung34/vagrant-ovirt3/master/example_box/ovirt.box HTTP/1.1
> User-Agent: Mozilla/5.0 (compatible; MSIE 9.0; Windows NT 6.1; Trident/5.0)
> Host: raw.githubusercontent.com
> Accept: */*
> Referer: https://raw.github.com/myoung34/vagrant-ovirt3/master/example_box/ovirt.box
> 
< HTTP/1.1 404 Not Found
< Date: Tue, 13 Jan 2015 10:42:22 GMT
* Server Apache is not blacklisted
< Server: Apache
< Access-Control-Allow-Origin: https://render.githubusercontent.com
< Content-Security-Policy: default-src 'none'
< X-XSS-Protection: 1; mode=block
< X-Frame-Options: deny
< X-Content-Type-Options: nosniff
< Strict-Transport-Security: max-age=31536000
< Content-Length: 9
< Accept-Ranges: bytes
< Via: 1.1 varnish
< X-Served-By: cache-ams4136-AMS
< X-Cache: MISS
< X-Cache-Hits: 0
< Vary: Authorization,Accept-Encoding
< Expires: Tue, 13 Jan 2015 10:47:22 GMT
< Source-Age: 0
< 
* Connection #1 to host raw.githubusercontent.com left intact
Not Found
```

**Comment by myoung34** [comment]
I don't think github puts binary files on raw.githubusercontent.com, so that link isn't right.
Should be: https://github.com/myoung34/vagrant-ovirt3/blob/master/example_box/dummy.box?raw=true

**Comment by myoung34** [comment]
Fixed in b770dc2
